release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -1792,6 +1795,8 @@ With this release, the CCO no longer checks for the nonexistent permission.
 
 * Previously, Cluster Version Operator (CVO) changes that were introduced in {product-title} 4.15.0, 4.14.0, 4.13.17, and 4.12.43 caused failing risk evaluations to block the CVO from fetching new update recommendations. When the risk evaluations failed, the bug caused the CVO to overlook the update recommendation service. With this release, the CVO continues to poll the update recommendation service, regardless of whether update risks are being successfully evaluated and the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-25708[*OCPBUGS-25708*])
 
+* Previously, the Cluster Version Operator (CVO) was unable to find and download release signatures for standalone clusters that were running `4.16.0-rc.3` and `4.16.0-rc.4` with a default feature set. With this release, the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-35249[*OCPBUGS-35249*])
```

Comment:
I actually asked Olivia to remove this RN from her PR because the bug was created in 4.16 and resolved before 4.16 became GA, meaning no customers will see this bug in production. I've also confirmed this with the SME, so we're good to remove this, and I'll set the Jira to Release Note Not Required.
```suggestion
```

Reply:
Thanks. Removed and Jira updated.